bentoml/server/bento_api_server.py

```diff
@@ -190,8 +191,12 @@ def bento_service_api_wrapper(api, service_name, service_version):
     """
     Create api function for flask route
     """
-    summary_name = str(service_name) + "_" + str(api.name)
-    request_metric_time = Summary(summary_name, summary_name + " request latency")
+    request_metric_time = Summary(
+        api.name,
+        api.name + " request latency",
+        namespace=service_name,
```

Comment:
maybe namespace is not the most appropriate choice.  It collides with our deployment's namespace, that could lead to confusion. Maybe change it to `bento_service_name`

Reply:
it's prometheus' namespace...

Reply:
I'd recommend use `"BENTOML"` as default namespace, and make it configurable(add to the config list). And it would be better to include service name as part of the metric name, so the metric name will become `{service_name}_{api.name}`. I think most commonly people are just using `predict`, `classify` as API name, and it can get confusing if they have multiple BentoService deployed.

Reply:
Ok. Just thought it would be helpful when users want to query based on service_name when the Prometheus is shared among different Bento Services. Anyway, we can revise this later, for now, I'll change back to `BENTOML`.